packages/@react-aria/table/docs/useTable.mdx

```diff
@@ -1080,24 +1075,47 @@ const Resizer = React.forwardRef((props, ref) => {
         height: 'auto',
         border: '2px',
         borderStyle: 'none solid',
-        borderColor: layoutState.resizingColumn === column.key ? 'orange' : 'grey',
+        borderColor: layoutState.resizingColumn === column.key || isFocusVisible ? 'orange' : 'grey',
```

Comment:
One problem with this approach: how to convey to the user the difference between the resizer having focus vs the resizer being "active"/movable via the arrow keys? At the moment there isn't a difference visually, not sure the best way to do so tbh since usually a slider is always "active"